docs/platform_operators/define-loadbalancer-firewalls.md

```diff
@@ -0,0 +1,51 @@
+# Configure Cloud Provider's Firewalls for Istio Loadbalancer
+
+## Objective
+
+At end of this setup, you will be able to install cf-for-k8s with a loadbalancer that is only accessible to clients with the specific IP addresses.
+
+## Prerequisites
+
+- Cluster should support `LoadBalancer` services.
+
+## Steps to setup cloud provider firewalls
+
+The following instructions assume you have created `cf-install-values.yml`.
+
+1. Add `istio_source_ranges` key and the IP ranges that are allowed to access the load balancer to your `cf-install-values.yml`
+
+    > Note that the append annotation must be applied to each item you want to insert.
+
+    ```yaml
+    istio_source_ranges:
+    #@overlay/append
+    - "130.211.204.1/32"
+    #@overlay/append
+    - "130.211.204.2/32"
+    ```
+
+1. Follow the instructions from deploy doc to generate the final deploy yml using `ytt` and then `kapp` deploy cf-for-k8s to your cluster.
```

Comment:
It'd be nice to provide a hyperlink to the deploy doc here

Reply:
Thank you for your feedback! I've updated PR accordingly.